cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	kbutil "github.com/operator-framework/operator-sdk/internal/util/kubebuilder"
+)
+
+//nolint:maligned
+type bundleCmd struct {
+	// Options to turn on different parts of bundling.
+	kustomize bool
+	manifests bool
+	metadata  bool
+
+	// Common options.
+	operatorName string
+	version      string
+	inputDir     string
+	outputDir    string
+	manifestRoot string
+	apisDir      string
+	crdsDir      string
+	stdout       bool
+	quiet        bool
+
+	// Metadata options.
+	channels       string
+	defaultChannel string
+	overwrite      bool
+}
+
+//nolint:lll
```

Comment:
Why `//nolint:lll`? This lint means that the line is too extended. Is it really required in this scenario?

Reply:
I'll be adding some links in the long description that will be over 118 characters :grimacing:.